### ISSUE TITLE
Feature request: git-blame each line and go to git graph

### ISSUE THREAD

**Issue by pgsandstrom** [phil294/git-log--graph#40]
My most common use case is this: I see some code in my editor. I want to see it in a visualized context. So currently I manually git-blame the line, and then search for the commit in the visualized git log. If your extension would offer an automation of this, it would be my personal killer feature.

Here is a git-blame extension under the MIT license: https://github.com/Sertion/vscode-gitblame

It has the git-blame feature that I want, I would just want to tie it in with a visualized git log like this.

**Comment by phil294** [comment]
Fully agree! I've been thinking that it would be useful to be integrated into GitLens's (hugely popular extension) blame view:

![image](https://github.com/phil294/git-log--graph/assets/14108705/b37343e8-8893-4764-a818-238e6bffe2aa)

but there appears to be no plugin api for said extension. But I guess it's easy to just ship our own git blame line tooling. How should it be accessible though? I'd like to support all of

- status bar (like Sertion/vscode-gitblame)
- context menu (right click) at a position in the editor
- register new command

maybe there's more?

**Comment by phil294** [comment]
> I understand that this is a big feature to implement

nah it was quite okay actually.

I released a first version of this, would be glad if you could test it out some time.

**Comment by phil294** [comment]
(I think I accidentally broke all icons with this release, will fix later)

**Comment by pgsandstrom** [comment]
The feature works great! Awesome! Im going to try it out for a while.

**Comment by Amanoki** [comment]
Sorry for asking here. I'm not sure if this is an issue or if I'm not understanding it right.
I noticed that the editor's context menu has a "Blame: Focus last commit at cursor position in graph view" option (I was wondering where it came from until I realized here that it comes from git-log-graph).
It doesn't seem to do anything when I click on it. Sounds like it should activate the graph view and highlight that specific commit but it doesn't do that.
I tried the command `git-log--graph.blame-line`, nothing reacted to it.
I have set `"git-log--graph.position": "view"` and the view is placed in the bottom panel of the editor. The line at cursor position does have a blame.
Other commands like `git-log--graph.start` can work but only this one doesn't work. Does it require any conditions or settings?
